### PR TITLE
Make the environment variables test correctly scope variables on a pe…

### PR DESCRIPTION
**Pull request by cpennington**
…r-job basis

### CODE REVIEW

File: edxpipelines/tests/scripts/test_scripts.py [hunk 1]
```diff
@@ -294,13 +295,27 @@ def global_environment_variables():
     yield 'PRIVATE_KEY'
 
 
+def environment_variables_provided_by(object):
+    """
+    Yield all environment variables provided by ``object``.abs
```
Comment (doctoryes) [inline edxpipelines/tests/scripts/test_scripts.py]: Nit: `abs`?

Comment (cpennington) [inline edxpipelines/tests/scripts/test_scripts.py]: Blarg. My autocomplete kicks off in documentation, so I often get `abs` at the end of sentences.

**Comment by rlucioni** [comment]
Belated 👍, though it looks like a fixup commit escaped unsquashed ಠ_ಠ.

**Comment by cpennington** [comment]
Oh, rats.

**Pull request merged**